### ISSUE TITLE
[ToDo] Finish patch-css package

### ISSUE THREAD

**Issue by matthew-dean** [jesscss/jess#20]
This is blocked by Rollup plugin / Webpack loader support. In order to do "live" theming, when this code is called:
```
import styles from './component.jess'

const css = styles()
```
... executing the `styles()` function should, if CSS injection (or "patching"?) was enabled, either inject or patch the stylesheets in the browser. See the explainer: https://github.com/jesscss/jess/tree/master/packages/patch-css

### Outstanding questions

1. Should simply calling the function patch CSS?
2. Should `styles()` be memo-ized such that we don't try to compare / patch at runtime if variables have not changed? (This might be good to do anyway for compile-time `@include`s). Should we memoize each input? Separate issue?

**Comment by matthew-dean** [comment]
This is done, although it needs more testing.
